### ISSUE TITLE
ValueError: 'FINISHED' is not a valid CalcJobState

### ISSUE THREAD

**Issue by yakutovicha** [aiidateam/aiida-tutorials#49]
When I try to generate aiida graph, I get the following error:

```
(aiida) max@ip-172-31-7-93:~$ verdi graph generate 1720
Traceback (most recent call last):
  File "/home/max/.virtualenvs/aiida/bin/verdi", line 10, in <module>
    sys.exit(verdi())
  File "/home/max/.virtualenvs/aiida/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/max/.virtualenvs/aiida/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/max/.virtualenvs/aiida/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/max/.virtualenvs/aiida/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/max/.virtualenvs/aiida/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/max/.virtualenvs/aiida/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/max/codes/aiida_core/aiida/cmdline/utils/decorators.py", line 72, in decorated_function
    return function(*args, **kwargs)
  File "/home/max/codes/aiida_core/aiida/cmdline/commands/cmd_graph.py", line 62, in generate
    include_calculation_outputs=outputs)
  File "/home/max/codes/aiida_core/aiida/tools/visualization/graphviz.py", line 127, in draw_graph
    nodes[inp.pk] = draw_node_settings(inp)
  File "/home/max/codes/aiida_core/aiida/tools/visualization/graphviz.py", line 72, in draw_node_settings
    additional_string = "\\n {}".format(node.get_description())
  File "/home/max/codes/aiida_core/aiida/orm/nodes/process/calculation/calcjob.py", line 543, in get_description
    return self.get_state()
  File "/home/max/codes/aiida_core/aiida/orm/nodes/process/calculation/calcjob.py", line 234, in get_state
    return CalcJobState(state)
  File "/home/max/.virtualenvs/aiida/lib/python3.6/enum.py", line 293, in __call__
    return cls.__new__(cls, value)
  File "/home/max/.virtualenvs/aiida/lib/python3.6/enum.py", line 535, in __new__
    return cls._missing_(value)
  File "/home/max/.virtualenvs/aiida/lib/python3.6/enum.py", line 548, in _missing_
    raise ValueError("%r is not a valid %s" % (value, cls.__name__))
ValueError: 'FINISHED' is not a valid CalcJobState
```

**Comment by odarbelaeze** [comment]
This might be fixed with #52

**Comment by sphuber** [comment]
> This might be fixed with #52

No, these are two separate problems, but I have created [a new updated archive](https://drive.google.com/open?id=1BKqZbMt-yMuZqbFD33zjkqtt59f6IwDM) which solves both problems. Please download and import it to verify if it solved the issues

**Comment by ltalirz** [comment]
fixed in #73
